meson.build

```
@@ -65,6 +65,10 @@ if compiler.get_argument_syntax()== 'gcc' # used for gcc compatible compilers
     link_args += ['-Wl,--enable-new-dtags'] # Set RUNPATH instead of RPATH
   endif
 
+  if host_machine.system()=='darwin'
+    link_args += ['-framework', 'CoreFoundation'] # Set RUNPATH instead of RPATH
```

maybe use a dependency instead of just the linker args, that should guarantee that the include paths are available as well.

```suggestion
    deps += dependency('appleframeworks', modules: ['foundation'])
```

---

Implemented in bc09cd87ffad6b894811579be3b20336f94407b1